SYCL/GroupAlgorithm/barrier.cpp

```diff
@@ -1,3 +1,6 @@
+// Temporarily disable:
+// XFAIL: *

```

Comment:
```suggestion
// Temporarily disabled due to regressions introduced by https://github.com/intel/llvm/pull/8412.
// REQUIRES: TEMPORARY_DISABLED

```
This is accepted idiom to disable the test.
Please, open bug reports for disabled tests to track the status of these issues.